tests/test_input_output.py

```diff
@@ -580,3 +580,121 @@ def test_io_SplineMagneticField(tmpdir_factory):
     for key in derivs1.keys():
         for key2 in derivs1[key].keys():
             np.testing.assert_allclose(derivs1[key][key2], derivs2[key][key2])
+
+
```

Comment:
adding a test where you compare the DESC Equilibrium quantities to quantities from the gfile directly would be good, maybe for some simple case (can't the SOLOVEV be ran with EFIT? DESC compares very well with that case)

Reply:
I don't have access to any Grad-Shafranov solvers currently, including the ones in EFIT.